pages/conference/young-inspirers.vue

```diff
@@ -0,0 +1,318 @@
+<template>
+    <i18n-page-wrapper class="px-8 sm:px-10 md:px-32 lg:px-60" custom-x>
+        <core-h1 title="2021 Young Inspirers TW" center></core-h1>
+        <div
+            class="h-40 sm:h-52 md:h-64 w-full mb-8"
+            :style="bannerStyle"
+        ></div>
+        <p class="intro">{{ intro }}</p>
+
+        <ul class="list-disc">
+            <li class="listTitle">活動時間：</li>
+            <p>2021/08/14 13:30-17:00</p>
+            <li class="listTitle">活動形式：</li>
+            <span>Youtube 線上直播：</span
+            ><ext-link href="https://youtu.be/SWnwRHgt-RA" highlight underline
+                >點我看直播</ext-link
+            ><br />
+            <span>線上共同筆記：</span
+            ><ext-link
+                href="https://hackmd.io/OXhLGazIRs6v4a_U_Zwq0g"
+                highlight
+                underline
+                >點我看共筆</ext-link
+            >
+        </ul>
```

Comment:
Too messy and should not place `<p>` at the same level as `<li>` here.

It should be something like this:

```suggestion
        <ul class="list-disc">
            <li class="listTitle">活動時間：</li>
            <li class="list-none">2021/08/14 13:30-17:00</li>
            <li class="listTitle">活動形式：</li>
            <li class="list-none">
                Youtube 線上直播：
                <ext-link
                    href="https://youtu.be/SWnwRHgt-RA"
                    highlight
                    underline
                    >點我看直播</ext-link
                >
            </li>
            <li class="list-none">
                線上共同筆記：
                <ext-link
                    href="https://hackmd.io/OXhLGazIRs6v4a_U_Zwq0g"
                    highlight
                    underline
                    >點我看共筆</ext-link
                >
            </li>
        </ul>
```